packages/browser/vite.config.ts

```diff
@@ -0,0 +1,36 @@
+import { resolve } from "path";
+import { defineConfig } from "vite";
+import dts from "vite-plugin-dts";
+
+import surveysPackageJson from "../surveys/package.json";
+import packageJson from "./package.json";
+
+const config = ({ mode }) => {
+  const isDevelopment = mode === "dev";
+  const formbricksSurveysScriptSrc = isDevelopment
+    ? "http://localhost:3003/index.umd.js"
+    : `https://unpkg.com/@formbricks/surveys@~${surveysPackageJson.version}/dist/index.umd.js`;
```

Comment:
Also the surveys package is currently still loaded from unpkg. This should also be loaded from the formbricks api endpoint :-)